composer.json

```
@@ -13,7 +13,7 @@
     },
     "require": {
         "php": ">=8.1",
-        "giggsey/libphonenumber-for-php": "^8.9",
+        "giggsey/libphonenumber-for-php": "^9.0.1",
```

```suggestion
        "giggsey/libphonenumber-for-php": "^9.0.2",
```

To make sure we can rely on PhoneNumberUtil::UNKNOWN_REGION again.